pkg/timestamp/microts.go

```diff
@@ -5,7 +5,7 @@ import (
 	"sync/atomic"
 	"time"
 
-	"github.com/golang/protobuf/ptypes/timestamp"
+	timestamp "google.golang.org/protobuf/types/known/timestamppb"
```

Comment:
Please use the package name without an alias.
```suggestion
	"google.golang.org/protobuf/types/known/timestamppb"
```

The same for `_test` file.